pkg/golinters/mnd.go

```diff
@@ -0,0 +1,61 @@
+package golinters
+
+import (
+	mnd "github.com/tommy-muehle/go-mnd/v2"
+	"golang.org/x/tools/go/analysis"
+
+	"github.com/golangci/golangci-lint/pkg/config"
+	"github.com/golangci/golangci-lint/pkg/golinters/goanalysis"
+)
+
+func NewMND(settings *config.MndSettings) *goanalysis.Linter {
```

Comment:
I thought MND means "monad" (like in functional programming) :D 
really not obvious that mnd is for "magic number detector"

why not use name like "magic-number"?

Reply:
We will not change the name of linters, golangci-lint is a wrapper.